ComponentKit/Core/CKComponent.mm

```diff
@@ -154,7 +154,7 @@ - (CKComponentViewContext)viewContext
     _mountInfo->viewContext = {v, {{0,0}, v.bounds.size}};
     return {.mountChildren = YES, .contextForChildren = effectiveContext.childContextForSubview(v, g.didBlockAnimations)};
   } else {
-    CKAssertNil(_mountInfo->view, @"Didn't expect to sometimes have a view and sometimes not have a view");
+    CKAssertNil(_mountInfo->view, @"Didn't expect %@ to have a mounted view: %@", [self class], [_mountInfo->view class]);
```

Comment:
I thought about including some information from the view configuration itself but figured this may be enough. I'd be open to including the identifier of the view configuration's view class as well if we think that would be helpful.